tests/Dockerfile.vic-integration-test

```diff
@@ -0,0 +1,15 @@
+# VIC Test Image
+#
+# Build a docker image that runs latest Ubuntu with golang, robotframework and vmware-vic installed
+# (https://golang.org/)
+# (http://robotframework.org/)
+# (https://github.com/vmware/vic)
+#
+# docker build -t vic-integration-test --no-cache -f Dockerfile.vic-integration-test .
+
+from vic-integration-test-base
+
+# Install latest version of vmware-vic
+run go get github.com/vmware/vic; echo 0
```

Comment:
Yea, this was unfortunately due to our infra problems - I couldn't hook into git properly with drone because of the infra problems, which would have allowed us to build per PR instead of just grabbing master. The infra issue might have been fixed this weekend, so I am going to see if I can get drone working off of PRs again.